preprocessor/preprocessor.py

```diff
@@ -258,6 +259,16 @@ def get_alignment(self, tier):
         start_time = 0
         end_time = 0
         end_idx = 0
+
+        new_tier = [] # MFA2.0(default config) may yield slices with no token which leads a mismatch in length.
+        for t1, t2 in zip(tier._objects[:-1], tier._objects[1:]):
+            new_tier.append(t1)
+            if (t2.start_time - t1.end_time) != 0:
+
+                warnings.warn('Find a slice with empty text.')
+                new_tier.append(tgt.Interval(start_time=t1.end_time, end_time=t2.start_time, text='spn'))
+        new_tier.append(t2)
```

Comment:
Why is new_tier not being used anywhere later in this code?

Reply:
While running I am getting multiple such warnings on a dataset, but is it really fixing the issue?